### DISCUSSION TITLE
Enable DevTools in production build? [v2]

### DISCUSSION THREAD

**Discussion by TechplexEngineer** [wailsapp/wails#1415] [Q&A] [answered]
Is there a way to enable devtools in production builds? 

I'm building an app where users will be writing custom javascript code and would benefit from access to the console and other devtools.

**Comment by leaanthony** [comment, marked as answer]
`wails build -debug`

**Reply by TechplexEngineer** [reply]
Thanks!

**Comment by garfieldbanks** [comment]
On Windows,  the "-debug" option also opens a console window. Is there a way to allow devtools but not have this console window open up?
